### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by CalderonJh** [community/community#109625] [GitHub Education] [answered]
### Hi everyone,

Hi, I was authenticated as a student about a month ago, but I don't know what I have to do to get the benefits of Student Developer Pack, like Copilot. Can you guys help me, please?

**Comment by CalderonJh** [comment, marked as answer]
Ok, if you see the All Offers section in https://education.github.com/pack you will see what you have to do to get each benefit xd

**Comment by gilsonpolito-etec** [comment]
Hello, do teachers have access to the "Student developer Pack"?

**Reply by CalderonJh** [reply]
I think not, for teachers there are other benefits such as GitHub Team and GitHub Classroom
